script/Arcueid/Day 10/10_00_ARC10_5E.txt

```diff
@@ -603,42 +658,47 @@ As she dangles the sword she carries in her hand, she sways and...
 [sha:02ff7880ad0d659c6e1561a7d2b801df2835fe37]{
 -- Page 669, Offset 26055. Mods: @k, @e.
 -- っていうかアナタ口悪すぎ！　お偉いさんのボンボンだからってあんまり調子にのる
-No, you've got way too foul a mouth! Just because you're a spoiled kid of some bigwig doesn't mean you can call me whatever youーーー
+No, you've got way too foul a mouth!
+ Just because you're the spoiled kid of some bigwig doesn't mean you can call me whatever you―――
 }
 [sha:ec30df41bfd11eb60bf87effde521b8d7479234f]{
 -- Page 669, Offset 26056. Glued. Mods: @x.
 -- ぴぎせゃあにゃーーーーーあ！！！！？」
- Pigiseyanyaaaa!!!!?"
+// Leading space for glue
+ %{i}Pigiseyanyaaaa!!!!?%{/i}"
 }
 [sha:fa2e282459ac8d5ac4846ef406c8d52db461e9b5]{
 -- Page 670, Offset 26057.
 -- 「……………、えっと」
-"Uh... well..."
+"Uh... what..."
 }
 [sha:1fdab5fffb174f2dd56f2397eb5ab834b6bd5b8a]{
 -- Page 670, Offset 26058.
 -- 修道服の女は不思議な叫び声をあげ、ひとりでにぴょんぴょんと跳ね回っている。
-The woman in nun's clothing lets out a peculiar scream and hops around on her own.
+The woman in nun's clothing lets out a peculiar scream and suddenly starts leaping around of her own accord.
 }
 [sha:05751708b577a923d744e817d3b303865725de14]{
 -- Page 670, Offset 26059.
 -- 先ほどの殺意も、地平線の彼方に消え去った。
-The murderous intent from earlier has vanished far beyond the horizon.
+// Understandable but kinda weird as a calque, proxied
```

Comment:
translator word of the day calendar paying off i see
<img width="48" alt="image" src="https://user-images.githubusercontent.com/16092757/163491138-1547a4c4-abc1-42ea-b49d-74a8a1a5bbef.png">

Reply:
I read that translation book and I _will_ use the knowledge I gained to be insufferable on the internet